src/components/components/member/member.js

```diff
@@ -392,9 +392,9 @@ class Member extends React.PureComponent {
 
     const github = isEditing ? (
       <div className="member__text-field--wrapper-with-icon">
-        <GitHub className="member__icon" />
+        <GitCommit className="member__icon" />
         <TextInput
-          label="Github Url"
+          label="Github/Gitlab/Bitbucket Url"
```

Comment:
Maybe it would be better if we wrote something like "Git server url (for example Github, Gitlab or Bitbucket)" ? 

@bacharakis thoughts?

Reply:
I'll update it to use that language... it's more specific